src/components/SchemaRow.tsx

```diff
@@ -90,6 +90,7 @@ export const SchemaPropertyRow: typeof SchemaRow = ({ node, onGoToRef, rowOption
 
       <div className="flex-1 flex truncate">
         <Property node={node} onGoToRef={onGoToRef} />
+        {metadata.schema.type && typeof metadata.schema.format === 'string' && <Format nodeSchema={metadata.schema} />}
```

Comment:
Let's use `schema` then. IIRC we use it in other spots when we pass schema around.